disnake/threads.py

```diff
@@ -371,6 +380,17 @@ def is_nsfw(self) -> bool:
         parent = self.parent
         return parent is not None and parent.is_nsfw()
 
+    def is_pinned(self) -> bool:
+        """Whether the thread is pinned in a :class:`ForumChannel`
+
+        Pinned threads do not get affected by the auto archive duration.
```

Comment:
are not affected

Reply:
> A thread can be pinned within a forum. A thread that is pinned will have the `(1 << 1)` flag set. Archiving a pinned thread will unset the flag. A pinned thread will not auto archive.

https://github.com/discord/discord-api-docs/commit/fba22cbd77f537c0d6efaa1b800ba58952a4355a#diff-e28250ee18796453c6332a572fb2fe1f25be18b46bbb56934789cfaef57edec2R193

Reply:
I meant we should change this line to `Pinned threads are not affected by the auto archive duration.`